inst/WORDLIST

```diff
@@ -259,6 +275,7 @@ getXParametersForDemogrpahyPlot
 getXParametersForPKParametersPlot
 getYParametersForDemogrpahyPlot
 getYParametersForPKParametersPlot
+Github

```

Comment:
please remove this one.
It should be `GitHub` with capital H, then it will be accepted